### PR TITLE
[Enhancement] Express example

### PR DESCRIPTION
**Pull request by macrigiuseppe**
This PR implements a Kepler.gl example using: Express, Webpack #664

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2963,7 +2963,7 @@ decamelize@^1.1.1, decamelize@^1.2.0:
   version "1.2.0"
   resolved "https://registry.yarnpkg.com/decamelize/-/decamelize-1.2.0.tgz#f6534d15148269b20352e7bee26f501f9a191290"
 
-deck.gl@^7.1.5:
```
Comment (heshan0131) [inline yarn.lock]: you don't have to do this in this PR. I have one out

File: examples/node-app/.babelrc [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    ["@babel/plugin-proposal-decorators", { "legacy": true }],
```
Comment (heshan0131) [inline examples/node-app/.babelrc]: do we need this here if the module is loading compiled kepler.gl code?

File: examples/node-app/.babelrc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "@babel/plugin-proposal-class-properties",
+    ["@babel/transform-runtime", {
```
Comment (heshan0131) [inline examples/node-app/.babelrc]: same here

File: examples/node-app/.babelrc (hunk 2 above)
Comment (macrigiuseppe) [reply examples/node-app/.babelrc]: I copied the babel config file from other examples but you were right. Those two options are not necessary

**Comment by macrigiuseppe** [comment]
@heshan0131 updated with your feedback

File: examples/node-app/package.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    "babel-plugin-module-resolver": "^3.0.0",
+    "babel-plugin-transform-builtin-extend": "^1.1.0",
+    "webpack": "^4.29.0",
+    "webpack-cli": "^3.2.1",
+    "webpack-dev-middleware": "^3.7.0",
+    "webpack-dev-server": "^3.1.14",
+    "webpack-hot-middleware": "^2.25.0",
+    "webpack-stats-plugin": "^0.2.1"
+  },
+  "resolutions": {
```
Comment (heshan0131) [inline examples/node-app/package.json]: delete resolutions

File: examples/node-app/package.json [hunk 5]
```diff
@@ -0,0 +1,53 @@
+{
+  "scripts": {
+    "start": "node server.js",
+    "start-local": "WEBPACK_ENV=es6 node server.js"
+  },
+  "dependencies": {
+    "express": "^4.17.1",
+    "global": "^4.3.0",
+    "kepler.gl": "^1.1.5",
```
Comment (heshan0131) [inline examples/node-app/package.json]: `^1.1.6`

Comment (macrigiuseppe) [reply examples/node-app/package.json]: I published version 1.1.6 and updated the file

File: examples/node-app/package.json (hunk 4 above)
Comment (macrigiuseppe) [reply examples/node-app/package.json]: Removed after updating kepler to 1.1.6

**Comment by macrigiuseppe** [comment]
@heshan0131 updated with your feedback

**Pull request merged**